distribution/lib/Standard/Database/0.0.0-dev/src/Connection/SSL_Mode.enso

```diff
@@ -0,0 +1,17 @@
+from Standard.Base import all
+
+type SSL_Mode
+    ## Do not use SSL for the connection.
+    type Disable
+
+    ## Prefer SSL for the connection, but does not verify the server certificate.
+    type Prefer
+
+    ## Will use SSL but does not verify the server certificate.
+    type Require
+
+    ## Will use SSL, validating the certificate but not verifying the hostname.
+    type Verify_CA ca_file:Nothing|File|Text=Nothing
```

Comment:
I think it would be worth explaining what happens if `ca_file` is set to `Nothing`. Is some default CA used instead?